l10n_nl_state/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "l10n_nl_state",
+    "version": "10.0.1.0.0",
+    "author": "Therp BV, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "Dutch Localization",
+    "summary": "",
+    "depends": [
+        'base',
+    ],
+    "data": [
+        'data/l10n_nl_states.xml',
+    ],
+   "installable": True,
```

Comment:
Line 17 and 18 are missing one space, so not aligned with rest. After improving this, squash all commits into one.